packages/fiori/src/TimelineItem.hbs

```diff
@@ -24,7 +23,7 @@
 				<slot></slot>
 			</div>
 		{{/if}}
-		<span class="ui5-tli-bubble-arrow ui5-tli-bubble-arrow--left"></span>
+		<span class="{{classes.arrowOrientation}}"></span>
```

Comment:
Use `classes.arrow`, not  specifying that the classes are about the orientation, but rather using more generic name - the classes for the timeline item arrow (later we can add other classes, that do not have anything to do with the layout, orientation thing).

Reply:
Done